### PR TITLE
Include cluster health job in openebs deploy stage

### PR DESCRIPTION
**Pull request by shashank855**

### CODE REVIEW

File: stages/openebs-install/cluster-2-health-check [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+echo "*************************Checking the Cluster's Health********************"
+
+echo "Checking for the number of nodes in ready state*******************************"
+ready_nodes=$(kubectl get nodes --no-headers | grep -v NotReady | wc -l)
+
+if [ "$ready_nodes" -eq 6 ]; then
+echo "Number of nodes in ready state is $ready_nodes"
+echo "******Cluster is in Healthy state****"
+
+echo "************Cloning openebs/e2e-test Repository****"
```
Comment (harshshekhar15) [inline stages/openebs-install/cluster-2-health-check:21]: @shashank855 let's also remove the e2e-test repo cloning from here -- https://github.com/mayadata-io/oep-e2e-rancher/blob/oep-release/stages/cluster-cleanup/cluster2-cleanup#L29

File: stages/openebs-install/cluster-2-health-check [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+git clone https://github.com/openebs/e2e-tests.git
+kubectl apply -f e2e-tests/hack/crds.yaml 
+kubectl apply -f  e2e-tests/hack/rbac.yaml
+kubectl apply -f utils/e2e-crd.yaml
+
+bash utils/e2e-cr jobname:cluster-2-health-check jobphase:Waiting
+bash utils/e2e-cr jobname:tcid-iuoi02-openebs-install jobphase:Waiting
+bash utils/e2e-cr jobname:tcid-ppcp01-create-cstor-pool jobphase:Waiting
+bash utils/e2e-cr jobname:cluster-2-health-check jobphase:Waiting
```
Comment (harshshekhar15) [inline stages/openebs-install/cluster-2-health-check]: jobphase will be `Running` here.

File: stages/openebs-install/tcid-iuoi02-openebs-install [hunk 3]
```diff
@@ -9,9 +9,9 @@ sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd oep-e2e-
...
 node() {
 
-bash utils/e2e-cr jobname:tcid-iuoi02-openebs-install jobphase:Waiting
+bash utils/pooling jobname:cluster-2-health-check
 bash utils/e2e-cr jobname:tcid-iuoi02-openebs-install jobphase:Running 
-bash utils/e2e-cr jobname:tcid-ppcp01-create-cstor-pool jobphase:Waiting
+
 
 # Use cluster2 kube config
 export KUBECONFIG=~/.kube/cluster2_config
```
Comment (harshshekhar15) [inline stages/openebs-install/tcid-iuoi02-openebs-install:12]: We need to export it before the above CR update commands as they are running on cluster2.

Comment (harshshekhar15) [reply stages/openebs-install/tcid-iuoi02-openebs-install:12]: We need to do the same for `tcid-ppcp01-create-cstor-pool` test case too.

File: stages/openebs-install/cluster-2-health-check (hunk 1 above)
Comment (shashank855) [reply stages/openebs-install/cluster-2-health-check:21]: I made that change.

File: stages/openebs-install/tcid-iuoi02-openebs-install (hunk 3 above)
Comment (shashank855) [reply stages/openebs-install/tcid-iuoi02-openebs-install:12]: This change exists.

**Review by harshshekhar15 — APPROVED** [review]
/lgtm

**Pull request merged**